addons/loyalty/static/src/scss/loyalty.scss

```diff
@@ -70,3 +50,15 @@
         height: fit-content;
     }
 }
+
+.o_nocontent_help:has(.loyalty-template) {
```

Comment:
On smaller screens the content overflows a bit the table header

<img width="944" height="897" alt="Image" src="https://github.com/user-attachments/assets/3d2adc11-3a03-4c24-8ecf-1748cca25ae8" />